examples/function-dot-graph.glu

```diff
@@ -40,38 +41,35 @@ let binary =
   | None -> error "Could not load binary"
 
 
-let function = 
-  match falcon.env "FUNCTION_NAME" with
-  | Some function_name ->
-    let find functions i name =
-      if i == (array.len functions) then
-        None
-      else
-        let function_entry = array.index functions i
-        match loader.function_entry.name function_entry with
-        | Some entry_name ->
-          if entry_name == name then
-            Some function_entry
-          else
-            find functions (i + 1) name
-        | None -> find functions (i + 1) name
-    let function_entries = loader.loader.function_entries binary
-    let function_entry = find function_entries 0 function_name
-    let function_entry =
-      match function_entry with
-      | Some function_entry -> function_entry
-      | None -> error "Failed to find function entry"
-    let address = loader.function_entry.address function_entry
-    loader.loader.function binary address
-  | None ->
-    match falcon.env "FUNCTION_ADDRESS" with
-    | Some address ->
-      let address =
-        match int.from_str_radix address 16 with
-        | Ok address -> address
-        | Err _ -> error "Failed to parse base16 address"
+let function =
```

Comment:
This introduces new behavior, which might be undesirable (basically, try parsing the argument as an int before treating the argument as a function name). I can revert the change for this script and/or the others if you'd prefer that.